### PR TITLE
feat(tutorials): tutorial for k8s video 3

### PR DESCRIPTION
**Pull request by RoRoJ**
### Your checklist for this pull request

- [x] Check that the commit messages match our requested structure.
- [x] Name your pull request according to the [contribution guidelines](../docs/CONTRIBUTING.md).

### Description
Creation of companion tutorial for third k8s video

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: tutorials/get-started-kubernetes-loadbalancer/index.mdx [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+In our previous tutorials, we created a [Kubernetes Kapsule](https://www.scaleway.com/en/kubernetes-kapsule/) and deployed the containerized [whoami](https://github.com/jwilder/whoami) application to it. This is what our cluster looked like at the end of the last tutorial:
+
+<Lightbox src="scaleway-cluster.webp" alt="" />
+
+## Why do we need a Load Balancer?
+
+By default, Kubernetes clusters are not exposed to the Internet. These means that external users cannot access the application deployed in our cluster.
+
+In the previous tutorial, we solved this problem by creating a **NodePort** Service for our cluster. NodePort is a Kubernetes service which opens a port on each node in a cluster that contains pods for the specified deployment. It then forwards any external traffic received on that port to the right pods. This allowed us to go to our node’s external IP address in a browser, specifying the open port, and be served  our web application.
```
Comment (nedaehm) [inline tutorials/get-started-kubernetes-loadbalancer/index.mdx:40]: I think something's missing after "be served"

File: tutorials/get-started-kubernetes-loadbalancer/index.mdx [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+## Step 1: Create Load Balancer from YAML manifest
+
+1. Open a terminal, and run the following command to check the state of your cluster:
+
+  ```
+  kubectl get all
+  ```
+
+  You should see two pods, the NodePport Service created during the previous tutorial (along with the basic default ClusterIP Service), the deployment and the two replicasets.
```
Comment (nedaehm) [inline tutorials/get-started-kubernetes-loadbalancer/index.mdx:83]:
```suggestion
  You should see two pods, the NodePport Service created during the previous tutorial (along with the basic default ClusterIP Service), the deployment and the two replica sets.
```

File: tutorials/get-started-kubernetes-loadbalancer/index.mdx [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+## Where were we?
+
+In our previous tutorials, we created a [Kubernetes Kapsule](https://www.scaleway.com/en/kubernetes-kapsule/) and deployed the containerized [whoami](https://github.com/jwilder/whoami) application to it. This is what our cluster looked like at the end of the last tutorial:
+
+<Lightbox src="scaleway-cluster.webp" alt="" />
+
+## Why do we need a Load Balancer?
+
+By default, Kubernetes clusters are not exposed to the Internet. These means that external users cannot access the application deployed in our cluster.
```
Comment (ldecarvalho-doc) [inline tutorials/get-started-kubernetes-loadbalancer/index.mdx:38]:
```suggestion
By default, Kubernetes clusters are not exposed to the Internet. This means that external users cannot access the application deployed in our cluster.
```

File: tutorials/get-started-kubernetes-loadbalancer/index.mdx (hunk 1 above)
Comment (ldecarvalho-doc) [inline tutorials/get-started-kubernetes-loadbalancer/index.mdx:40]: the sentence I changed was a bit long, but maybe my suggestion doesnt make sense
```suggestion
In the previous tutorial, we solved this problem by creating a **NodePort** Service for our cluster. NodePort is a Kubernetes service which opens a port on each node in a cluster. This cluster contains pods for the specified deployment. It then forwards any external traffic received on that port to the right pods. This allowed us to go to our node’s external IP address in a browser, specifying the open port, and be served  our web application.
```

Comment (RoRoJ) [reply tutorials/get-started-kubernetes-loadbalancer/index.mdx:40]: thanks, fixed

Comment (RoRoJ) [reply tutorials/get-started-kubernetes-loadbalancer/index.mdx:40]: This suggestion changes the meaning, but I've tried to improve the sentence a different way.

**Review by ldecarvalho-doc — APPROVED** [review]

**Review by nedaehm — APPROVED** [review]

**Pull request merged**